plugin-annotation/src/main/java/com/mapbox/mapboxsdk/plugins/annotation/CircleOptions.java

```diff
@@ -223,6 +223,18 @@ public CircleOptions withLatLng(LatLng latLng) {
     return this;
   }
 
+  /**
+   * Get the LatLng of the circle, which represents the location of the circle on the map
+   *
+   * @return the location of the circle in a longitude and latitude pair
+   */
+  public LatLng getLatLng() {
+    if (geometry == null) {
+      return null;
+    }
+    return new LatLng(geometry.latitude(), geometry.longitude());
```

Comment:
I feel like the `LatLng` value should be cached next to the `geometry` so that frequent invocations do not lead to unnecessary allocations. What do you think @tobrun?

Reply:
Same for all the other getters.

Reply:
this seems like double bookkeeping and that would require updating the LatLng, List<LatLng>, .. when you update the geometry. This kinda defeats the proposed optimization. My idea is to have the geometry api as fast as possible (animation would be impacted) and  expose the LatLng API for ease of use (where speed doesn't matter else use geometry api instead). thoughts?

Reply:
> animation would be impacted

True, after all, I think it's more beneficial to have fast setter rather than getter. There's always a fallback to raw `geometry` to increase getter performance.